README.md

```diff
@@ -1826,27 +1828,30 @@ _You can enable the following settings in Xcode by running [this script](resourc
     case neptune = 7
   }
 
+  // WRONG
   enum ErrorCode: Int {
     case notEnoughMemory
     case invalidResource
     case timeOut
   }
 
   // RIGHT
+  // Relying on Swift's automatic enum values
   enum ErrorType: String {
     case error
     case warning
   }
 
+  // RIGHT
   /// These are written to a logging service. Explicit values ensure they're consistent across binaries.
-  // swiftformat:disable redundantRawValues
```

Comment:
We should keep this example linter directive, since otherwise the linter would remove the redundant raw values

Reply:
@calda Thank you for your explanation! I learned something new today! I have fixed accordingly. Please review again. 😄